src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1482,4 +1482,13 @@
   <data name="PreferStringContainsOverIndexOfCodeFixTitle" xml:space="preserve">
     <value>Replace with 'string.Contains'</value>
   </data>
+  <data name="DetectPLINQNopsDescription" xml:space="preserve">
+    <value>'AsParallel' does not have an effect if it is the last Expression or the only remaining Expressions are 'ToList' or 'ToArray'.</value>
```

Comment:
```suggestion
    <value>'AsParallel' has no effect if it is the last expression in chain or if the only remaining expressions are 'ToList' or 'ToArray'.</value>
```

Reply:
Maybe we want to be a little more generic on the last part. I mean that instead of only listing `ToList` and `ToArray` we could say having a `the remaining expression is a collection creation`. Because I assume that `ToDictionary`, `ToLookup` are having the same behavior.